ydb/core/security/certificate_check/cert_auth_processor.cpp

```diff
@@ -98,6 +99,15 @@ TVector<std::pair<TString, TString>> X509CertificateReader::ReadIssuerTerms(cons
     return ReadTerms(name);
 }
 
+TString X509CertificateReader::GetFingerprint(const X509Ptr& x509) {
+    unsigned char fingerprint[128];
```

Comment:
is there any corresponding constant for that?

Reply:
Yes, I added constant SHA_DIGEST_LENGTH for sha1 digest